src/js/decafmud.display.standard.js

```diff
@@ -66,6 +66,25 @@ var Display = function(decaf, ui, disp) {
 		e.preventDefault();
 	});
 	
+	// Function to check screen length vs display length
+	this.truncateHeight = function() {
+	    if ((this.display.clientHeight < (window.innerHeight * this.decaf.options.set_display.maxscreens)) || (this.display.children.length < this.decaf.options.set_display.minelements)) {
+	        return;
+	    }
+	    height = 0;
+	    elems = [];
+	    while (height < (this.display.clientHeight - (window.innerHeight * this.decaf.options.set_display.maxscreens))) {
```

Comment:
Let's store `this.display.clientHeight - (window.innerHeight * this.decaf.options.set_display.maxscreens)` in a variable since it's computed N times right now.